doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -185,7 +185,8 @@ This section describes the core set of services and service components needed to
 
 <a name="3.3.2"></a>
 ### 3.3.2. Tenant Isolation
-Content to be developed
+In OpenStack user interfaces and documentation, a group of users is referred to as a project or tenant. With Keystone v1 and v2(both deprecated), the term "tenant" was used. With Keystone v3, the term "project" got adopted and both the terms became interchangeable. However, as CNTT RA uses Keystone v3 in [this](chapter05.md#5.2) section, so it is recommended to use the term "project" when referring to OpenStack and use [tenant](../../ref_model/chapters/chapter03.md#321-tenant) when referring to multi-tenancy . According to [OpenStack glossary](https://docs.openstack.org/doc-contrib-guide/common/glossary.html), Projects represent the base unit of "ownership" in OpenStack, in that all resources in OpenStack should be owned by a specific project.
```

Comment:
OpenStack Glossary
user: In OpenStack Identity, entities represent individual API consumers and are owned by a specific domain. In OpenStack Compute, a user can be **associated** with roles, projects, or both.

 "In OpenStack user interfaces and documentation, a group of users is referred to as a project or tenant." -- can we delete the sentence as it causes unnecessary confusion as a project is defined later in the paragraph.

Reply:
agreed

Reply:
Ok